plugins/kotlin/completion/tests/testData/handlers/basic/AmbiguousSuperMethodWithArgument.kt

```diff
@@ -0,0 +1,18 @@
+// FIR_COMPARISON
+package foo.bar
+interface A {
+    fun foo(i: Int) {}
+}
+
+interface B {
+    fun foo(i2: Int) {}
+}
+
+class C : A, B {
+    override fun foo(j: Int) {
+        super.fo<caret>
+    }
+}
+
+// ELEMENT: "foo"
```

Comment:
What does this test check?

Reply:
This is to make sure the type qualifier is present when completing super member with call args.